app/assets/javascripts/discourse/app/widgets/widget-dropdown.js

```diff
@@ -199,15 +200,19 @@ export const WidgetDropdownClass = {
     return { id: attrs.id };
   },
 
-  defaultState() {
+  defaultState(attrs) {
     return {
       opened: false,
+      disabled: (attrs.options && attrs.options.disabled) || false,
     };
   },
 
   buildClasses(attrs) {
     const classes = ["widget-dropdown"];
     classes.push(this.state.opened ? "opened" : "closed");
+    if (this.state.disabled) {
```

Comment:
can we keep the same style as the line above please? (I don't mind this style, but I would prefer consistency).